### PR TITLE
renamed the whitesource stable product name

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: project/Whitesource.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ object Whitesource extends AutoPlugin {
         if (isSnapshot.value)
           if (gitCurrentBranch.value == "master") "master"
           else "adhoc"
-        else "stable"
+        else "2.5-stable"
```
Comment (patriknw) [inline project/Whitesource.scala]: would be better to extract this from the `version`. Do we do that in some other place?

Comment (raboof) [reply project/Whitesource.scala]: We originally did that for whitesource ;), but then moved to this version scheme to be able to easily report on the 'latest supported' versions. Let me see if I can find an example.

Comment (raboof) [reply project/Whitesource.scala]: `def majorMinor(version: String): Option[String] ="""\d+\.\d+""".r.findFirstIn(version)`

Comment (raboof) [reply project/Whitesource.scala]: so something like `majorMinor(version.value).map(_ + "-stable").getOrElse("snapshot"))` (not sure whether 'snapshot' makes sense here anymore though)

**Comment by raboof** [comment]
Perhaps also update to sbt-whitesource 0.1.6, as that includes error handling for when the upload fails.

File: project/Whitesource.scala (hunk 1 above)
Comment (patriknw) [reply project/Whitesource.scala]: thanks, I was thinking we could keep `master` and `adhoc` without version info

**Comment by patriknw** [comment]
@raboof updated, please review again

**Comment by patriknw** [comment]
failure is probably because of failed update, please take a look @raboof

**Comment by raboof** [comment]
Hmm no this looks like https://github.com/typesafehub/sbt-whitesource/issues/20

**Comment by patriknw** [comment]
@raboof I removed the version update

**Review by raboof — APPROVED** [review]

**Pull request merged**